solidity/dashboard/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dashboard",
-  "version": "1.3.4",
+  "version": "1.4.0",
```

Comment:
Should we have the version in `package-lock.json` updated as well?

Reply:
Oof. Let's… Leave it as-is, I don't think it has a material impact even on `npm ci`.